src/components/JobSummary/Summary/Summary.js

```diff
@@ -18,8 +20,25 @@ const extractStats = (summary) => {
     }
 }
 
-const Summary = ({ summary, importType }) => {
-    // gml import type object return
+const Summary = ({ summary, importType, isDryRun }) => {
+    if (isDryRun && importType === 'TRACKER_IMPORT_JOB') {
```

Comment:
I would just put this in a variable to make the intention clearer
```suggestion
   const isTrackerDryRun = isDryRun && importType === 'TRACKER_IMPORT_JOB
    if (isTrackerDryRun) {
```

and maybe add a comment for posterity:

```
// the new tracker endpoints don't return a full summary, so we're just showing the ignored/total
```